src/toast/ops/sim_tod_atm.py

```diff
@@ -539,14 +504,18 @@ def _get_scan_range(self, obs, comm, prefix):
         # Read the extent of the AZ/EL boresight pointing, and use that
         # to compute the range of angles needed for simulating the slab.
 
-        # FIXME:  This is quite arbitrary and assumes that the boresight
-        # pointing has been created by the SimGround operator.  We should
-        # define the observation keys to use in some standard place and
-        # compute these limits from the data if they do not exist.
-        min_az_bore = obs["scan_min_az"].to_value(u.radian)
-        max_az_bore = obs["scan_max_az"].to_value(u.radian)
-        min_el_bore = obs["scan_min_el"].to_value(u.radian)
-        max_el_bore = obs["scan_max_el"].to_value(u.radian)
```

Comment:
Thanks for fixing this.  Now I can remove a workaround in another downstream package.